openai/components/asset.yaml

```diff
@@ -0,0 +1,2 @@
+type: component
+spec: oai_finetune_pipeline_component.yaml
```

Comment:
Please rename oai_finetune_pipeline_component.yaml to spec.yaml to keep it generic after the files are moved under the subdir.

Reply:
ok